src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/Person.kt

```diff
@@ -15,5 +15,23 @@ data class Person(
     example = "Morgan",
     type = "string",
   )
+  val lastName: String,
+
+  val aliases: List<Alias> = listOf()
+)
+
+data class Alias(
```

Comment:
Would it be easier to find this Alias model if it lived in a dedicated file?

Reply:
Yuppppp, agreed!